aten/src/ATen/native/Blas.cpp

```diff
@@ -14,6 +14,9 @@ scalar_t dot_impl(int64_t n, scalar_t *x, int64_t incx, scalar_t *y, int64_t inc
 template<typename scalar_t>
 scalar_t vdot_impl(int64_t n, scalar_t *x, int64_t incx, scalar_t *y, int64_t incy);
 
+template<typename scalar_t>
+void addr_impl(int64_t m, int64_t n, scalar_t alpha, scalar_t *x, int64_t incx, scalar_t *y, int64_t incy, scalar_t *a, int64_t lda);
+
 constexpr inline bool lda_cond(int64_t m, int64_t n, int64_t lda) {
   return n == 1 || lda > std::max<int64_t>(1L, m);
```

Comment:
`lda_cond` function is not re-used in this migration, as it uses `greater than` instead of `greater than or equal to` to compare  `lda` with `std::max<int64_t>(1L, m)`, which is not consistent with the old TH code like below.
```cpp
#define LDA_COND(M, N, LDA) ((N) == 1 || (LDA) >= THMax(1, (M)))
```

I tried to change `>` to `>=`, the test cases of `test_jit.py` would fail due to the `nan` numbers.
One typical example(`test_addmv_broadcast_lhs_coef_cpu - test_jit.TestJitGeneratedAutogradCPU`) is below
```
  File "/opt/conda/lib/python3.6/site-packages/torch/testing/_internal/common_utils.py", line 1153, in assertEqual
    exact_dtype=exact_dtype, exact_device=exact_device)
  File "/opt/conda/lib/python3.6/site-packages/torch/testing/_internal/common_utils.py", line 1127, in assertEqual
    self.assertTrue(result, msg=msg)
AssertionError: False is not true : Tensors failed to compare as equal! With rtol=1e-07 and atol=1e-07, found 1 element(s) (out of 10) whose difference(s) exceeded the margin of error (including 1 nan comparisons).
The greatest difference was nan (nan vs. -2.1490962697169906), which occurred at index 2.
```

I decided to port `LDA_COND` with a new method `lda_cond_addr` in terms of this pull request.
We may further investigate why #30898 ported `lda_cond` this way.

Reply:
Let's start by asking @zasdfgbnm.

Reply:
I don't think I did so for a reason. It could be I forgot a `=` from `<=` and luckily avoided an issue.